### PR TITLE
docs - DATA-SCHEMA -> DATA_SCHEMA

### PR DESCRIPTION
**Pull request by lisakowen**
docs for https://github.com/greenplum-db/pxf/pull/954.

notes:
- we already removed THREAD-SAFE from the docs.  
- the docs do not reference either STATS-SAMPLE-RATIO or STATS-MAX-FRAGMENTS, so the only option of interest is really DATA-SCHEMA.  updated the docs to use the name with underscore.

(i will include this name change in the release notes.  i will also add DATA-SCHEMA to the deprecated features section of the release notes.)

### CODE REVIEW

**Review by denalex — APPROVED** [review]
Are we going to add a deprecation notice somewhere else ?

**Comment by lisakowen** [comment]
sure, i'll add a deprecation notice on the two affected pages.

File: docs/content/hdfs_seqfile.html.md.erb [hunk 1]
```diff
@@ -68,7 +68,7 @@ You specify the compression type and codec, and the Java serialization/deseriali
 |-------|-------------------------------------|
 | COMPRESSION_CODEC    | The compression codec alias. Supported compression codecs include: `default`, `bzip2`, `gzip`, and `uncompressed`. If this option is not provided, Greenplum Database performs no data compression. |
 | COMPRESSION_TYPE    | The compression type to employ; supported values are `RECORD` (the default) or `BLOCK`. |
-| DATA-SCHEMA    | The name of the writer serialization/deserialization class. The jar file in which this class resides must be in the PXF classpath. This option is required for the `hdfs:SequenceFile` profile and has no default value. |
+| DATA_SCHEMA    | The name of the writer serialization/deserialization class. The jar file in which this class resides must be in the PXF classpath. This option is required for the `hdfs:SequenceFile` profile and has no default value. (**Note**: The equivalent option named `DATA-SCHEMA` is deprecated.) |
```
Comment (bradfordb-vmware) [inline docs/content/hdfs_seqfile.html.md.erb:71]: Do you we want to state more directly that deprecated items may be removed in a future release?

> **Note**: The equivalent option named `DATA-SCHEMA` is deprecated and may be removed in a future release.

**Review by bradfordb-vmware — COMMENTED** [review]
Accidentally repeated myself.

**Review by bradfordb-vmware — APPROVED** [review]

**Review by Aayushs4 — APPROVED** [review]

**Review by yimingli-vmware — APPROVED** [review]

**Pull request merged**